js/apps/admin-ui/src/identity-providers/add/OIDCGeneralSettings.tsx

```diff
@@ -19,9 +20,14 @@ export const OIDCGeneralSettings = ({ id }: { id: string }) => {
     formState: { errors },
   } = useFormContext();
 
+  const [aliasText, setAliasText] = useState(id);
```

Comment:
There should be no need to keep this in state, since React Hook Form will already keep this value in the form state. You should be able to get a live version of this state using [`useWatch()`](https://www.react-hook-form.com/api/usewatch/).